.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,84 @@
+        'class_attributes_separation' => ['elements' =>['property' => 'only_if_meta', 'const' => 'only_if_meta']],
+        'combine_consecutive_issets' => true,
+        'combine_consecutive_unsets' => true,
+        'comment_to_phpdoc' => true,
+        'date_time_immutable' => true,
+        'declare_strict_types' => true,
+        'doctrine_annotation_array_assignment' => true,
+        'doctrine_annotation_braces' => true,
+        'doctrine_annotation_indentation' => true,
+        'doctrine_annotation_spaces' => true,
+        'escape_implicit_backslashes' => true,
+        'explicit_indirect_variable' => true,
+        'explicit_string_variable' => true,
+        'final_internal_class' => false,
+        'general_phpdoc_annotation_remove' => false,
+        'header_comment' => false,
+        'heredoc_to_nowdoc' => false,
+        'linebreak_after_opening_tag' => false, // allows for <?php declare(strict_types=1);
+        'list_syntax' => ['syntax' => 'short'],
+        'mb_str_functions' => true,
+        'method_chaining_indentation' => true,
+        'multiline_comment_opening_closing' => true,
+        'multiline_whitespace_before_semicolons' => true,
+        'native_function_invocation' => false,
+        'no_alias_functions' => true,
+        'no_blank_lines_before_namespace' => false,
+        'no_extra_blank_lines' => true,
+        'no_null_property_initialization' => true,
+        'no_php4_constructor' => true,
+        'no_superfluous_elseif' => true,
+        'no_superfluous_phpdoc_tags' => true,
+        'no_unreachable_default_argument_value' => true,
+        'no_useless_else' => true,
+        'no_useless_return' => true,
+        'not_operator_with_space' => false,
+        'not_operator_with_successor_space' => false,
+        'ordered_class_elements' => false,
+        'php_unit_dedicate_assert' => false,
+        'php_unit_expectation' => false,
+        'php_unit_method_casing' => ['case' =>'snake_case'],
+        'php_unit_mock' => false,
+        'php_unit_namespaced' => false,
+        'php_unit_no_expectation_annotation' => false,
+        'php_unit_set_up_tear_down_visibility' => true,
+        'php_unit_strict' => false,
+        'php_unit_test_annotation' => false,
+        'php_unit_test_class_requires_covers' => false,
+        'phpdoc_add_missing_param_annotation' => true,
+        'phpdoc_order' => true,
+        'phpdoc_order_by_value' => true,
+        'phpdoc_to_return_type' => true,
+        'phpdoc_types_order' => ['null_adjustment' => 'always_last'],
+        'pow_to_exponentiation' => true,
+        'psr_autoloading' => true,
+        'random_api_migration' => false,
+        'simplified_null_return' => true,
+        'single_line_throw' => false,
+        'static_lambda' => false,
+        'strict_comparison' => true,
+        'strict_param' => true,
+        'string_line_ending' => true,
+        'ternary_to_null_coalescing' => true,
+        'void_return' => true,
+    ]);
```

Comment:
got some more options in mine:

```
        'array_syntax' => ['syntax' => 'short'],
        'backtick_to_shell_exec' => true,
        'blank_line_before_statement' => true,
        'class_keyword_remove' => false,
        'compact_nullable_typehint' => true,
        'date_time_immutable' => true,
        'fully_qualified_strict_types' => true,
        'no_alternative_syntax' => true,
        'ordered_imports' => true,
        'phpdoc_to_comment' => false,
        'yoda_style' => true,

```

Reply:
those are covered in `@Symfony` ruleset

Reply:
`class_keyword_remove` is marked as deprecated without successor, should we keep it regardless?

Reply:
it's ok  then